scripts/copy-files.js

```diff
@@ -15,6 +16,16 @@ function copyFile(file) {
   }).then(() => console.log(`Copied ${file} to ${buildPath}`));
 }
 
+function copyTypings(folder) {
```

Comment:
Why adding some extra logic when instead, we can put the definitions in the `/src` folder?
I'm worried about about having a more complex than needed publish script.